plugins/rapid7_insight_agent/icon_rapid7_insight_agent/util/graphql_api/api_connection.py

```diff
@@ -350,3 +446,16 @@ def _get_agents_from_result_object(self, results_object: dict) -> [dict]:
             )
 
         return agent_list
+
+    def _convert_hostnames_to_id(self, hostnames: List[str]) -> List[str]:
+        """
+        Function designed for the `quarantine_multiple` action which converts
+        the array of hostnames to an array of agent IDs
+        :param hostnames: Array containing the hostnames as a string
+        :return: Array containing host IDs
+        """
+        agent_list = []
+        for agent in hostnames:
+            agent_list.append(self._get_agent_id(agent))
+
+        return agent_list
```

Comment:
It could be written in more pythonic way, but it's up to you 😄 
```suggestion
        return [self._get_agent_id(agent) for agent in hostnames]
```

Reply:
Would it be okay to leave this as it is just for better readability?

Reply:
Yup